src/app/Http/Requests/RegisterRequest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Requests;
+
+
+use Illuminate\Validation\Rule;
+use Illuminate\Foundation\Http\FormRequest;
+
+
+class RegisterRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
```

Comment:
do you think this method authorized is being used??

Reply:
Already removed this method Sir, thank you!